public/js/chessClasses/gameManager.js

```diff
@@ -2,25 +2,43 @@ class GameManager {
   constructor() {
     this.selected = false
     this.legalSpaces = []
-    this.turn = 1
+    this.turn = WHITE
     this.board = new Board( 8, 8 )
-    this.board.setSquare( 'c2', new Piece())
-    this.board.setSquare( 'e7', new Piece( KNIGHT, BLACK ) )
+    this.placeStartingPositions()
   }
 
   onClick( coord ) {
     const piece = this.board.getSquare( coord )
+    const clickedMyPiece = this.clickedMyPiece( piece )
 
-    if( !piece && !this.selected ){
+    if( !clickedMyPiece && !this.selected ){
```

Comment:
If this code is intended to represent the case that nothing has been clicked and nothing has been selected, why would it need to be executed in an `onClick` handler?  This may be another opportunity to improve the naming of the function?